src/components/InputTodo.js

```diff
@@ -1,108 +0,0 @@
-import React from 'react'
-import TodoList from './TodoList'
-import {useState} from 'react'
-
-export default function InputTodo() {
-  const [Editable,setEditable] = useState(-1)
-  const [data,setData] = useState([]);
-  const [todo,settodo] = useState({});
-  const [title,setTitle] = useState("");
-  const [duedate,setDuedate] = useState("");
-  const [desc,setDesc] = useState("");
-  const [addUpdate,setAddUpdate] = useState(1);
-  const AddTask = ()=>{
-    if(todo?.title === undefined) {alert('Enter Todo Title'); return -1;}
```

Comment:
if(todo?.title === undefined) {alert('Enter Todo Title'); return -1;}
 this can be written as:
 if(todo?.title ) {alert('Enter Todo Title'); return -1;}